encoding/rs/extend_poly.go

```diff
@@ -1,19 +1,16 @@
-package cpu
+package rs
 
 import (
-	"github.com/Layr-Labs/eigenda/encoding"
 	"github.com/Layr-Labs/eigenda/encoding/fft"
 	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
 )
 
-type RsCpuComputeDevice struct {
+type RsDefaultComputeDevice struct {
```

Comment:
Can "default" be made specific? Like RsGnarkComputeDevice (suppose it's based on that)

Reply:
i think if we replaced protolambdas FFT implementation that we are borrowing with the gnark FFT then it would make sense to be called gnark ebcause then the FFT and MSM are both from the gnark library.

Reply:
"default" is not quite informative for what this does. Also if icicle becomes the primary backend, this as the default doesn't look well.

The argument here seems that it's not purely Gnark but a mixed use of libraries. If we plan to move away from protoblambda's FFT, it may be fine to preliminarily call it Gnark? But open to other good way to name it.

Reply:
I'm happy with calling it Gnark backend